modules/ROOT/pages/index.adoc

```diff
@@ -5,71 +5,19 @@ endif::[]
 :imagesdir: ../assets/images/
 :page-aliases: general:getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc
 
-Anypoint Exchange provides the benefit of being able to discover, share, and incorporate assets and resources into your applications. Anypoint Exchange helps you create API developer portals, view and test APIs, simulate data to APIs (Mocking Service), create assets, and use API Notebooks to describe and test API functions.
+Anypoint Exchange is your curated catalog of APIs and other assets, including API groups, API specification fragments, policies, and more.
```

Comment:
Can we elaborate a bit more on the value and specify assets beyond just APIs as well. Something on the lines of "Anypoint Exchange is a catalog of reusable assets such as APIs, Policies, API spec fragments, API groups as well as other integration assets such as connectors, templates, examples and custom assets. Anypoint Exchange can be used to catalog (publish), share, discover, learn about, and reuse assets within your organization, to drive collaboration, boost productivity and promote standards."

Reply:
done